src/primitives/transaction.cpp

```diff
@@ -248,12 +254,23 @@ unsigned int CTransaction::CalculateModifiedSize(unsigned int nTxSize) const
 std::string CTransaction::ToString() const
 {
     std::string str;
-    str += strprintf("CTransaction(hash=%s, ver=%d, vin.size=%u, vout.size=%u, nLockTime=%u)\n",
-        GetHash().ToString().substr(0,10),
-        nVersion,
-        vin.size(),
-        vout.size(),
-        nLockTime);
+    if (nVersion == 1 || nVersion == 2) {
+        str += strprintf("CTransaction(hash=%s, ver=%d, vin.size=%u, vout.size=%u, nLockTime=%u)\n",
+            GetHash().ToString().substr(0,10),
+            nVersion,
+            vin.size(),
+            vout.size(),
+            nLockTime);
+    } else if (nVersion == 3) {
```

Comment:
`} else if (fOverwintered && nVersion == 3) {` per https://github.com/zcash/zips/pull/133#issuecomment-361616008

Reply:
This and the above comment are now simplified to:
```
if (!fOverwintered) { ... } else if (nVersion == 3) { ... }
```